runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -512,6 +512,10 @@ public void requestFocus()
 		{
 			OSXUtil.requestFocus();
 		}
+		else if (OSType.getOSType() == OSType.Windows && System.getProperty("os.name").contains("10"))
```

Comment:
Why only on windows 10?

Reply:
Also does this still work? I wanted to use .toFront together with the rest of stuff you had before

Reply:
Ok I will just do a check for all windows versions.